src/js/index.js

```diff
@@ -7,36 +7,37 @@ function getDiscussTopics() {
 	if (!section || !discussions) {
 		return;
 	}
+	let idList = [];
```

Comment:
Small optimization, but could you use a Set instead? Also rename the variable to something more descriptive:

```suggestion
	let latestTopicIds = new Set();
```